src/main/g8/src/main/scala/$package$/api/$if(enable_zio_http.truthy)$ApiZioHttp.scala$endif$

```diff
@@ -81,14 +88,14 @@ object ApiZioHttp {
         effect.foldZIO(handleError, _.toResponseZIO)
 
       case Method.DELETE -> !! / "items" / itemId =>
-        val effect: IO[DomainError, String] = for {
+        val effect: IO[DomainError, Int] = for {
```

Comment:
It doesn't make sense to return Int here. I think the type should be `IO[DomainError, Unit]`

Reply:
Thanks, fixed